__tests__/app.test.js

```diff
@@ -203,10 +206,17 @@ describe('app routes', () => {
       .expect('Content-Type', /json/)
       .expect(200);
 
+      console.log(`updated sticker request =${JSON.stringify(updatedSticker.body)}`);
+      //ask about whether this is doing the same thing as the updated sticker thing. This is making a get request, is using the put request return for the expected bypassing hitting th endpoint making this solution incorrect?
```

Comment:
The part above is the return from the update endpoint. This current step is an extra validation step, hitting the database again and confirming that the item in question was in fact changed.